spras/analysis/ml.py

```diff
@@ -44,7 +44,7 @@ def summarize_networks(file_paths: Iterable[Union[str, PathLike]]) -> pd.DataFra
             edges = []
             for line in lines:
                 parts = line.split('\t')
-                if len(parts) > 0:  # in case of empty line in file
+                if len(parts) >= 4:  # in case of empty line in file or line doesn't include all values
```

Comment:
Which case were we checking for here? Blank lines at the end of the file?

If a line has 2 parts, such as nodes but no rank or direction, should we raise an error instead of continuing?

Reply:
I added a check if a line in one of the files has > 4 or < 4 values in a line, and raise an error